sources/packages/web/src/components/common/notes/Notes.vue

```diff
@@ -108,9 +107,10 @@ export default {
   setup(props: any, context: any) {
     const { dateOnlyLongString, timeOnlyInHoursAndMinutes } = useFormatters();
     const showModal = ref(false);
-    const createNotesModal = ref({} as ModalDialog<void>);
+    const createNotesModal = ref({} as ModalDialog<NoteAPIInDTO | boolean>);
     const addNewNote = async () => {
-      await createNotesModal.value.showModal();
+      const addNewNoteData = await createNotesModal.value.showModal();
+      if (addNewNoteData) emitToParent(addNewNoteData as NoteAPIInDTO);
```

Comment:
pls use `{}`

Reply:
i think you can remove `emitToParent` function and move `context.emit("submitData", data);` here